OsmAnd/src/net/osmand/plus/activities/MapActivityActions.java

```diff
@@ -318,6 +318,8 @@ public void contextMenuPoint(final double latitude, final double longitude, fina
 				R.drawable.ic_action_search_light).reg();
 		adapter.item(R.string.context_menu_item_share_location).icons(
 				R.drawable.ic_action_gshare_dark, R.drawable.ic_action_gshare_light).reg();
+		adapter.item(R.string.context_menu_item_view_location).icons(
```

Comment:
I wouldn't support idea of having extra item in the location menu list, I would rather prefer to have as a sub selection on share location dialog, next QR code or so.